terraform/environments/analytical-platform-compute/secrets.tf

```
@@ -206,3 +206,26 @@ module "airflow_connections_slack_api_default_secret" {
 
   tags = local.tags
 }
+
+module "dashboard_service_app_secrets" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+  #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
+
+  count = terraform.workspace == "analytical-platform-compute-test" ? 0 : 1
+
+  source  = "terraform-aws-modules/secrets-manager/aws"
+  version = "1.3.1"
+
+  name        = "dashboard-service/app-secrets"
+  kms_key_id  = module.common_secrets_manager_kms.key_arn
+
+  secret_string = jsonencode({
+    secret_key          = random_password.dashboard_service_secret_key.result,
```

```suggestion
    secret_key          = random_password.dashboard_service_secret_key[0].result,
```

Adding index syntax as that resource now uses a count